lib/ueberauth/strategy/amco/oauth.ex

```diff
@@ -10,9 +10,9 @@ defmodule Ueberauth.Strategy.Amco.OAuth do
 
   @defaults [
     strategy: __MODULE__,
-    site: "https://oidc.amco.com",
-    token_url: "https://oidc.amco.com/oauth/token",
-    authorize_url: "https://oidc.amco.com/openid/authorize"
+    site: "https://idp.amco.me",
+    token_url: "https://idp.amco.me/oauth/token",
+    authorize_url: "https://idp.amco.me/openid/authorize"
```

Comment:
These should probably be relative or concatenated to the domain name. Hardcoding these here limits them only to the one system with no ability to change the targets if needed for staging, testing, troubleshooting etc. Since this is a public project, we should not have what might be our internal processes in public view. The domain should probably be a configuration option with the endpoints concatenated here.

Reply:
@brlanier This can be done by configuration. These are just the defaults. An example of Ueberauth Facebook: https://github.com/ueberauth/ueberauth_facebook/blob/master/lib/ueberauth/strategy/facebook/oauth.ex#L16
Or Ueberauth google:
https://github.com/ueberauth/ueberauth_google/blob/master/lib/ueberauth/strategy/google/oauth.ex

Configuration will look like this:
```elixir
config :ueberauth, Ueberauth.Strategy.Amco.OAuth,
  site: "SITE",
  token_url: "https://stage-idp.amco.me/oauth2/token",
  authorize_url: "https://stage-idp.amco.me/oauth2/authorize",
  client_id: "CLIENT_ID",
  client_secret: "CLIENT_SECRET"
```

This is how Ueberauth API works as an standard.

Reply:
@brlanier I made some improvements about it. Im closing this issue in favor of this one #4